### ISSUE TITLE
how to update content without restart http server like flask's debug mode?

### ISSUE THREAD

**Issue by CoderOverflow** [huge-success/sanic#1072]
app.run(host='0.0.0.0', port=8000, debug=True)
It doesn't work in sanic 0.7.0

After I modify my python files I have to restart server every time.
can debug mode support no restart update.

**Comment by r0fls** [comment]
Currently not supported. There's an open PR here: https://github.com/channelcat/sanic/pull/1047

**Comment by RafRaf** [comment]
I use [AoikLiveReload](https://github.com/AoiKuiyuyou/AoikLiveReload). Here is my code snippet:

```python
if __name__ == '__main__':
    # Use live reload in DEBUG = True
    #
    debug = settings.DEBUG  # My settings object

    if debug:
        from aoiklivereload import LiveReloader
        LiveReloader().start_watcher_thread()

    # Run the App
    #
    app.run(host='0.0.0.0', port=8000, debug=debug, workers=settings.WORKERS, access_log=debug)
```

**Comment by sjsadowski** [comment]
PR was merged prior to 0.8.0
